docs/rules/avoid-dispatching-multiple-actions-sequentially.md

```diff
@@ -1 +1,60 @@
 # avoid-dispatching-multiple-actions-sequentially
+
+> This rule promotes disallows dispatching multiple actions one after another.
+
+## Rule Details
+
+An action should be an event which abstracts away the details of store internals.
+An action can be a composition of several events, in which case the developer might be tempted to dispatch several actions in sequence. But a better approach is t dispatch one "combining" action, which exactly describes what that event entails, and then map it to several other actions in an Effect
+
+Examples of **incorrect** code for this rule:
+
+```ts
+export class Component implement OnInit {
+  constructor(
+    private readonly store: Store<AppState>,
```

Comment:
```suggestion
    private readonly store: Store,
```

Reply:
Wait, really?  Never realized this is a bad practice. How come? Is it possibly because some parts of the state might not be loaded?

Reply:
Yea, I'll create the docs for it and send you the link 😉